src/Microsoft.DotNet.ProjectModel/DependencyContextBuilder.cs

```diff
@@ -0,0 +1,34 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Threading.Tasks;
+using Microsoft.DotNet.ProjectModel;
+using Microsoft.DotNet.ProjectModel.Compilation;
+
+namespace Microsoft.Extensions.DependencyModel
+{
+    public static class DependencyContextBuilder
+    {
+        public static  DependencyContext FromLibraryExporter(LibraryExporter libraryExporter, string target, string runtime)
+        {
+            var dependencies = libraryExporter.GetDependencies();
```

Comment:
I'm pretty sure this leaves out the root project. I think we actually want the root project to be in the `DependencyContext`. We should use `GetAllExports` instead.